### PR TITLE
Gitlab metrics export

### PR DESCRIPTION
**Pull request by dan-mm**

### CODE REVIEW

File: eco-ci-gitlab.yml [hunk 1]
```diff
@@ -36,14 +36,29 @@ cache:
         - echo 'running eco-ci measure script'
         - |
             repo=$CI_PROJECT_NAMESPACE/$CI_PROJECT_NAME
-            /tmp/eco-ci/main/scripts/make_measurement.sh -l "$ECO_CI_LABEL" -r $CI_PIPELINE_ID -b $CI_COMMIT_REF_NAME -R $repo -c $CI_COMMIT_SHA -sd $ECO_CI_SEND_DATA -s "gitlab"
+            /tmp/eco-ci/main/scripts/make_measurement.sh \
```
Comment (ArneTR) [inline eco-ci-gitlab.yml:39]: why is -c $CI_COMMIT_SHA not on a separate line?

**Comment by ArneTR** [comment]
looks ok to me. The logic I could not fully check, but any changes look valid

**Comment by ArneTR** [comment]
Please merge if ready

File: eco-ci-gitlab.yml (hunk 1 above)
Comment (dan-mm) [reply eco-ci-gitlab.yml:39]: I missed that one, thanks

**Pull request merged**